.evergreen/config.yml

```diff
@@ -1476,46 +1370,22 @@ functions:
           cp $PROJECT_DIRECTORY/.evergreen/run-mongodb-aws-ecs-test.sh $ECS_SRC_DIR/.evergreen
```

Comment:
Suggest removing the `sed` line above. I expect it is no longer needed. Adding `TestData = {};\n` does not appear in the [C driver config](https://github.com/mongodb/mongo-c-driver/blob/5e5784d814b2e3912c2855ac2b9a172267b8235f/.evergreen/scripts/run-aws-tests.sh#L116-L133).

Reply:
updated